apps/platform-app/src/app/[locale]/(profile)/(routes)/profiles/layout.tsx

```diff
@@ -9,10 +11,12 @@ export default async function ProfilesLayout({
 }: {
   children: React.ReactNode
 }) {
+  const { user } = await getAuthorizedUser()
+  const t = await getTranslations(I18nNamespaces.Index)
   return (
     <main>
```

Comment:
**issue**: Ensure that the translation key 'visitorBaner' is correctly spelled as 'visitorBanner' to avoid potential translation errors.